index.html

```diff
@@ -1,19 +1,115 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
+    <header class="header">
+        <img src="file:///C:/Users/New%20User/Documents/GitHub/HTML-CSS-Coursework-Week1/HTML-CSS-Coursework-Week2/favicon.ico" alt="logo">
+        <nav class="nav-bar">
+            <ul>
+                <a>meet karma</a>
+                <a>how its works</a>
+                <div id="red"><a>store</a></div>
+                <a>blog</a>
+                <a>help</a>
+                <a>login</a>
+
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+        <div class="wifi">
+            <section class="karma">
+                <h1>interducing karma</h1>
+                <p>bring WIFI with you, Everywhere you go.</p>
+                <button class="learn">learn more</button>
+            </section>
+        </div>
 
+        <div class="h3">
+            <h1>Everyone need a little karma</h1>
```

Comment:
```suggestion
            <h2>Everyone need a little karma</h2>
```
We should only have one `h1`, so we can change this to `h2` instead